### DISCUSSION TITLE
uFiber instant

### DISCUSSION THREAD

**Discussion by cobz2022** [Anime4000/RTL960x#275] [General]
hi, i'm using uFiber instant, and i had a headache
when i tried switch to original fw, i forgot to erase mtd3. 
and now i stuck in ssh/telnet login username/pwd. 

does anyone know, how to solve this?

**Comment by Anime4000** [comment]
I would suggest buy ODI stick instead, solving would be issue as not dumping it first

**Comment by alexojegu** [comment]
Try: https://github.com/stich86/UF-Instant-Mod?tab=readme-ov-file#factory-reset-after-10-boot

> If you screw up the mtd3 partition with wrongs values, doing a fast swap for ten times (avoiding full Linux OS boot) will erase config partition and after OS boot, all parameters return to factory defaults
